# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libgli/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libgli-0.4/src/gli_glf.cpp.vanilla	2008-08-10 16:20:14.000000000 +0200
+++ libgli-0.4/src/gli_glf.cpp	2008-08-10 16:20:20.000000000 +0200
@@ -23,6 +23,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <math.h>
+#include <string.h>
 
 #ifdef MACOSX
 #include <OpenGL/gl.h>
--- libgli-0.4/src/gli_window.cpp.vanilla	2008-08-10 16:20:28.000000000 +0200
+++ libgli-0.4/src/gli_window.cpp	2008-08-10 16:20:40.000000000 +0200
@@ -17,6 +17,7 @@
 	Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 */
 #include <stdio.h>
+#include <string.h>
 #include "gli_window.h"
 #include "gli_button.h"
 #include "string"
